README.md

```diff
@@ -1,3 +1,121 @@
 <img src="icons/forklift-logo-lightbg.svg" alt="Logo" width="100" />
 
 # OpenShift Console Plugin For Forklift
+
+[![Operator Repository on Quay](https://quay.io/repository/yaacov/forklift-console-plugin/status "Plugin Repository on Quay")](https://quay.io/repository/yaacov/forklift-console-plugin)
+
+Forklift is a suite of migration tools that facilitate the migration of VM workloads to [OpenShift Virtualization](https://cloud.redhat.com/learn/topics/virtualization/).
+
+## Prerequisites
+
+* [__Forklift Operator__](https://https://github.com/konveyor/forklift-operator/)
+* [__OpenShift Console 4.11+__](https://www.openshift.com/)
+
+## Development
+
+[ See the [docs](/docs) folder for more information ]
+
+### Start forklift operator server without authentication:
+
+Start the [forklift-operator](https://github.com/konveyor/forklift-controller) inventory server on `http://localhost:8080` withut authentication.
+
+Using forklift-operator development code, login into the cluster and run the server without authentication:
+
+1. `oc login` (requires [oc](https://console.redhat.com/openshift/downloads) and an [OpenShift cluster](https://console.redhat.com/openshift/create))
+2. `AUTH_REQUIRED=false make run`
```

Comment:
This will not work with `OpenShift cluster` but with the `minikube` and even with it we need some extra steps like https://github.com/konveyor/forklift-operator/pull/239

Reply:
removed this section from the readme